x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichMetadata.java

```diff
@@ -0,0 +1,239 @@
+        );
+
+        static {
+            PARSER.declareString(ConstructingObjectParser.constructorArg(), TYPE);
+            PARSER.declareString(ConstructingObjectParser.constructorArg(), SOURCE_INDEX);
+            PARSER.declareString(ConstructingObjectParser.constructorArg(), QUERY_FIELD);
+            PARSER.declareStringArray(ConstructingObjectParser.constructorArg(), DECORATE_FIELDS);
+        }
+
+        private final Type type;
+        private final String sourceIndex;
+        private final String queryField;
+        private final List<String> decorateFields;
+
+        public Policy(StreamInput in) throws IOException {
+            this(Type.read(in.readString()), in.readString(), in.readString(), in.readStringList());
+        }
+
+        public Policy(Type type, String sourceIndex, String queryField, List<String> decorateFields) {
+            this.type = type;
+            this.sourceIndex = sourceIndex;
+            this.queryField = queryField;
+            this.decorateFields = decorateFields;
+        }
+
+        public Type getType() {
+            return type;
+        }
+
+        public String getSourceIndex() {
+            return sourceIndex;
+        }
+
+        public String getQueryField() {
+            return queryField;
+        }
+
+        public List<String> getDecorateFields() {
+            return decorateFields;
+        }
+
+        @Override
+        public void writeTo(StreamOutput out) throws IOException {
+            out.writeString(type.toString());
+            out.writeString(sourceIndex);
+            out.writeString(queryField);
+            out.writeStringCollection(decorateFields);
+        }
+
+        @Override
+        public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+            builder.field(TYPE.getPreferredName(), type.toString());
+            builder.field(SOURCE_INDEX.getPreferredName(), sourceIndex);
+            builder.field(QUERY_FIELD.getPreferredName(), queryField);
+            builder.array(DECORATE_FIELDS.getPreferredName(), decorateFields.toArray(new String[0]));
+            return builder;
+        }
+
+        @Override
+        public boolean isFragment() {
+            return true;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) return true;
+            if (o == null || getClass() != o.getClass()) return false;
+            Policy policy = (Policy) o;
+            return type == policy.type &&
+                sourceIndex.equals(policy.sourceIndex) &&
+                queryField.equals(policy.queryField) &&
+                decorateFields.equals(policy.decorateFields);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(type, sourceIndex, queryField, decorateFields);
+        }
+
+        public enum Type {
+
+            STRING;
```

Comment:
This type of policy will allow to do exact lookups for enrichment purposes (domain, user, organization entity lookups use cases). In the future we will add range and geo types as well.